### DISCUSSION TITLE
SCADENZA INSERIMENTO VALUTAZIONI VERIFICHE

### DISCUSSION THREAD

**Discussion by sempliciniluigi89** [iisgiua/giuaschool#95] [Idee e proposte]
Buongiorno ,per legge l'inserimento delle valutazioni deve avvenire entro  e non oltre un determinato numero di giorni dallo svolgimento delle stesse  ,passati questi  giorni non è più possibile inserire né valutazione orale né scritta  o di qualsiasi altro tipo.Si potrebbe implementare questa funzione dal lato segreteria in modo da dare un libero arbitrio sui giorni oltre i quali non è più possibile inserire le valutazioni?

**Comment by trinko** [comment]
In futuro è prevista la possibilità di bloccare le modifiche al registro dopo un certo numero di giorni, configurabile.
In questo modo, non solo le valutazioni, ma anche l'inserimento delle firme e altro dovrà essere effettuato entro X giorni dalla data di riferimento.
Ma non è una priorità e non so quando verrà realizzata.
